Makefile

```diff
@@ -172,10 +171,9 @@ clean:
 distclean:
 	@rm -rf node_modules
 
-# create list of translations, saved as `./calypso-strings.php`
+# create list of translations, saved as `./calypso-strings.pot`
 translate: node_modules $(CLIENT_CONFIG_FILE)
-	@CALYPSO_ENV=stage $(BUNDLER)
-	@CALYPSO_ENV=stage $(LIST_ASSETS) | xargs $(I18N_CALYPSO) --format php --output-file ./calypso-strings.php --array-name calypso_i18n_strings
+	@CALYPSO_ENV=stage $(I18N_CALYPSO) --format pot --output-file ./calypso-strings.pot $(JS_FILES)
```

Comment:
Interesting observation, though determining this is possibly outside the scope of this PR.